ports/esp32/modules/inisetup.py

```diff
@@ -33,8 +33,10 @@ def fs_corrupted():
 def setup():
     check_bootsec()
     print("Performing initial setup")
-    uos.VfsLfs2.mkfs(bdev)
-    vfs = uos.VfsLfs2(bdev)
+    # uos.VfsLfs2.mkfs(bdev)
```

Comment:
Please do not change default `LittleFS` to `FAT`, as file is shared among all ESP32 variants (freeze in [manifest.py](https://github.com/micropython/micropython/blob/master/ports/esp32/boards/manifest.py#L1)).
Best way would be to have the `inisetup.py` file override in board folder with a custom `manifest.py`, just like in `UM_TINYS2` board's [manifest.py](https://github.com/micropython/micropython/blob/master/ports/esp32/boards/UM_TINYS2/manifest.py#L2) file